### PR TITLE
add BigInt values as bind params

### PR DESCRIPTION
**Pull request by nicbaz**
Enable `scala.math.BigInt` values binding.

### CODE REVIEW

**Comment by nicbaz** [comment]
closed and reopened the PR to rerun the tests (network issue).

File: scalikejdbc-core/src/main/scala/scalikejdbc/ScalaBigIntegerConverterImplicits.scala [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package scalikejdbc
+
+import scala.language.implicitConversions
+
+/**
+ * Implicit conversions for BigInteger values.
+ */
+trait ScalaBigIntegerConverterImplicits {
```
Comment (seratch) [inline scalikejdbc-core/src/main/scala/scalikejdbc/ScalaBigIntegerConverterImplicits.scala]: I think it's ok to enable this trait by default as same as BigDecimal's one. Or is there any reason we should avoid to do so? https://github.com/scalikejdbc/scalikejdbc/blob/2.3.4/scalikejdbc-library/src/main/scala/scalikejdbc/package.scala#L52

**Comment by seratch** [comment]
Thank you for your great pull request. Supporting BigInt looks reasonable to me. :+1:

File: scalikejdbc-core/src/main/scala/scalikejdbc/ScalaBigIntegerConverterImplicits.scala (hunk 1 above)
Comment (nicbaz) [inline scalikejdbc-core/src/main/scala/scalikejdbc/ScalaBigIntegerConverterImplicits.scala]: oops missed that one, I've added the trait.

**Comment by seratch** [comment]
LGTM :+1: Thank you for your contribution :tada:

**Comment by xuwei-k** [comment]
There is a `implicit def javaBigInteger2bigInt(x: BigInteger): BigInt` in Scala.

https://github.com/scala/scala/blob/v2.11.7/src/library/scala/math/BigInt.scala#L105

Is `ScalaBigIntegerConverter` necessary?

**Comment by seratch** [comment]
@xuwei-k Thanks. Indeed, duplicated implicit conversion looks unnecessary. 
@nicbaz I'll revert only the trait later. I ask for your understanding.

**Comment by nicbaz** [comment]
@seratch sure no worries. Would you like me to do it?

**Comment by seratch** [comment]
@nicbaz I've not done yet. Yes, I will appreciate it if you do it too.

**Pull request merged**